hack/e2e/bmo-e2e-net.xml

```diff
@@ -1,15 +1,15 @@
 <network>
-  <name>baremetal-e2e</name>
+  <name>bmo-e2e</name>
   <forward mode='nat'>
     <nat>
       <port start='1024' end='65535'/>
     </nat>
   </forward>
-  <bridge name='metal3'/>
+  <bridge name='bmo-e2e'/>
   <ip address='192.168.222.1' netmask='255.255.255.0'>
     <dhcp>
       <range start='192.168.222.2' end='192.168.222.199'/>
-      <host mac='52:54:00:6c:3c:01' name='minikube' ip='192.168.222.199'/>
+      <host mac='52:54:00:6c:3c:01' name='ironic-0' ip='192.168.222.199'/>
```

Comment:
Why change this? The host is minikube, not Ironic. Ironic just happens to be running in minikube and using this address.

Reply:
Thanks. Since I didn't see any usage of this name, I was thinking naming it like this to make it less easy to mess up the ironic.env file. I'm not too keen on it though.

Reply:
Please change it back. This represents the hostname of whatever is using this IP (minikube). Ironic just happens to run in minikube also.